components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1246,6 +1258,19 @@ AuthenticatedUser buildAuthenticatedUser(String tenantAwareUser, String tenantDo
         return user;
     }
 
+    void validateAuthorizationCodeValidityPeriodConfiguration(OAuthConsumerAppDTO oAuthConsumerAppDTO) {
+
+        if (oAuthConsumerAppDTO.getAuthorizationCodeValidityPeriod() == 0) {
```

Comment:
How can we guarantee a 'null' value might not come? Is it handled from oAuthConsumerAppDTO level?

Reply:
`authorizationCodeValidityPeriod` is an instance variable of type `long` inside the class `OAuthConsumerAppDTO`. Therefore it is initialized to 0.

Reply:
ok